ui/conftest.py

```diff
@@ -1,194 +1,247 @@
-    if isinstance(required_extensions, str):
-        required_extensions = [required_extensions]
-    for ext in required_extensions:
-        source = (
-            libs.extract_tar_gz(
-                CHROME_TAR_PATH / f"{ext}.extension.tar.gz",
-                CHROME_DATA_PATH / "plugins",
-            )
-            / ext
-        )
-        if not result_path:
-            result_path = source
-        else:
-            result_path = result_path / f",{source}"
-    yield result_path
-    libs.rm_tree(CHROME_DATA_PATH)
-
-
-@pytest.fixture(scope="function", autouse=True)
-def chrome_extension_user_data() -> pathlib.Path:
-    """Extracting Chrome extension user data"""
-    user_data = (
-        libs.extract_tar_gz(CHROME_TAR_PATH / "user_data.tar.gz", CHROME_DATA_PATH)
-        / "user_data"
-    )
-    yield user_data
-    libs.rm_tree(user_data)
-
-
-@pytest.fixture(scope="session")
-def chrome_extension_password() -> str:
-    """Chrome extensions password `1234Neon5678`"""
-    try:
-        return os.environ["CHROME_EXT_PASSWORD"]
-    except KeyError:
-        raise AssertionError(
-            "Please set the `CHROME_EXT_PASSWORD` environment variable (password for wallets)."
-        )
-
-
-@pytest.fixture
-def use_persistent_context() -> bool:
-    """Flag used for Chrome extensions load, set to False for standard pages not extensions"""
-    return True
-
-
-# @pytest.hookimpl(tryfirst=True, hookwrapper=True)
-# def pytest_runtest_makereport(item, call):
-#     """Save screenshot on fail"""
-#     outcome = yield
-#     rep = outcome.get_result()
-#     if rep.when == 'call' and rep.failed:
-#         mode = 'a' if os.path.exists('failures') else 'w'
-#         try:
-#             with open('failures', mode):
-#                 if 'page' in item.fixturenames:
-#                     page = item.funcargs['page']
-#                 else:
-#                     print('Fail to take screenshot')
-#                     return
-#             allure.attach(
-#                 page.screenshot(full_page=True),
-#                 name='screenshot',
-#                 attachment_type=allure.attachment_type.PNG,
-#                 extension="png"
+from selenium import webdriver
+from ui.pages.developer_page import DeveloperPage
+from ui.pages.ecosystem_page import EcosystemPage
+from ui.pages.external_pages import ExternalPages
+from ui.pages.main_page import MainPage
+from ui.pages.menu import Menu
+from utils.base_class import BaseClass
+
+website_url = "https://neonevm.org/"
+
+# from ui import libs
+#
+# PLATFORM_NETWORKS = {
+#     "night-stand": "NEON EVM night-stand",
+#     "devnet": "NeonEVM DevNet",
+# }
+#
+# CHROME_TAR_PATH = pathlib.Path(__file__).absolute().parent / "extensions" / "data"
+# CHROME_DATA_PATH = (
+#     pathlib.Path(__file__).absolute().parent.parent / "chrome-data" / uuid.uuid4().hex
+# )
+# """CHROME_DATA_PATH is temporary local destination in project to untar chrome data directory and plugins"""
```

Comment:
is this necessary?

Reply:
Kristina asked don't delete this file

Reply:
@e-b-e-d-a but why you need to comment it
It is better to leave as it was if you don't need this changes for you tests

Reply:
I've uncommented this code